### DISCUSSION TITLE
Change config location on startup

### DISCUSSION THREAD

**Discussion by A-Lamia** [houmain/keymapper#180] [Q&A] [answered]
Hey,
Just wandering if it's possible to have a way of setting a custom config location on start up, currently i need to use the command line to change it every time i reset my PC thanks!

**Comment by ristomatti** [comment]
In which operating system?

**Reply by A-Lamia** [reply]
windows, i like to keep everything in `$HOME/.config` it's just easier to have in a single location, couple programs i use on windows work with XDG_CONFIG_HOME, that is real nice.

**Reply by ristomatti** [reply]
I wasn't aware XDG dirs are a thing on Windows. On Linux Keymapper's config is stored under `XDG_CONFIG_HOME`. Sorry can't help with Windows workarounds.

**Reply by houmain** [reply, marked as answer]
I also did not think of searching these variables' paths on Windows. I will try to add these in the next version.

You could also change the commandline of keymapper in the Windows Task Scheduler and append `-c "path to config"`.
![grafik](https://github.com/user-attachments/assets/a58dea2a-89da-4655-8e1b-927c2d359ef3)

**Reply by A-Lamia** [reply]
Yeah I've done this, adding XDG would be awesome, i multi-boot linux and windows so being able to have everything in a single place is a life saver.

**Reply by houmain** [reply]
`$XDG_CONFIG_HOME` and `$HOME/.config` are now also searched on Windows (in 4.9.0).
